test/specs/anInitialSuite.e2e.ts

```diff
@@ -96,34 +88,27 @@ describe('An Initial Suite', async () => {
   step('Set up the testing environment', async () => {
     testSetup = new TestSetup('AnInitialSuite', false);
     // Don't call testSetup.setUp() b/c we don't need to authorize a scratch org,
-    // just call setUpTestingEnvironment() and createInitialProject().
+    // just call setUpTestingEnvironment() and createProject().
     await testSetup.setUpTestingEnvironment();
-    await testSetup.createInitialProject('Standard');
+    await testSetup.createProject('Standard');
```

Comment:
`createProject()` takes an argument of `utilities.OrgEdition`, which has the options of `developer` or `enterprise`.

```suggestion
    await testSetup.createProject('developer');
```